doc/source/robust_estimators.md

```diff
@@ -1,17 +1,21 @@
-(robuststats)=
+(robust-estimators)=
 
-# The need for robust statistics
+# Need for robust estimators
 
-Digital elevation models often contain outliers that can be traced back to instrument acquisition or processing artefacts, and which hamper further analysis.
+Elevation data often contain outliers that can be traced back to instrument acquisition or processing artefacts, and which hamper further analysis.
 
-In order to mitigate their effect, xDEM integrates [robust statistics](https://en.wikipedia.org/wiki/Robust_statistics) at different levels:
-- Robust optimizers for the fitting of parametric models during {ref}`coregistration` and {ref}`biascorr`,
-- Robust measures for the central tendency (e.g., mean) and dispersion (e.g., standard deviation), to evaluate DEM quality and converge during {ref}`coregistration`,
-- Robust measures for estimating spatial autocorrelation for uncertainty analysis in {ref}`spatialstats`.
+In order to mitigate their effect, the analysis of elevation data can integrate [robust statistics](https://en.wikipedia.org/wiki/Robust_statistics) at different levels:
+- **Robust estimators for the central tendency (e.g., mean) and dispersion (e.g., standard deviation)**, to evaluate DEM quality and converge during {ref}`coregistration`,
```

Comment:
do you mean "median" and "NMAD" instead of mean and std?

Reply:
"converge" -> "convergence"? I am not sure if you mean "evaluate convergence" or something else

Reply:
It's true it was a bit convoluted, reworded!